README.md

```diff
@@ -94,29 +94,40 @@ ffmpeg -f lavfi -i \
 
 ## Installing TorchCodec
 
-Note: if you're on MacOS, you'll need to [build from source](./CONTRIBUTING.md).
-The instructions below assume you're on Linux.
+1. Install the latest stable version of PyTorch following the
+   [official instructions](https://pytorch.org/get-started/locally/). For other
+   versions, refer to the table below for compatibility between versions of
+   `torch` and `torchcodec`.
 
-  1. Install the latest stable version of PyTorch following the
-     [official instructions](https://pytorch.org/get-started/locally/). TorchCodec
-     requires [PyTorch 2.4](https://pytorch.org/docs/2.4/).
+2. Install FFmpeg, if it's not already installed. Linux distributions usually
+   come with FFmpeg pre-installed. TorchCodec supports all major FFmpeg versions
+   in [4, 7].
 
-  2. Install FFmpeg, if it's not already installed. Your Linux distribution probably
-     comes with FFmpeg pre-installed. TorchCodec supports all major FFmpeg versions
-     in [4, 7].
+   If FFmpeg is not already installed, or you need a more recent version, an
+   easy way to install it is to use `conda`:
 
-     If FFmpeg is not already installed, or you need a later version, install it with:
+   ```bash
+   conda install ffmpeg
+   # or
+   conda install ffmpeg -c conda-forge
+   ```
 
-     ```bash
-     conda install ffmpeg
-     # or
-     conda install ffmpeg -c conda-forge
-     ```
-  3. Install TorchCodec:
+3. Install TorchCodec:
 
-     ```bash
-     pip install torchcodec
-     ```
+   ```bash
+   pip install torchcodec
+   ```
+
+The following table indicates the compatibility between versions of
+`torchcodec`, `torch` and Python.
+
+| `torchcodec`       | `torch`            | Python              |
+| ------------------ | ------------------ | ------------------- |
+| `main` / `nightly` | `main` / `nightly` | `>=3.9`, `<=3.12`   |
+| not yet supported  | `2.5`              | `>=3.9`, `<=3.12`   |
+| `0.0.3`            | `2.4`              | `>=3.8`, `<=3.12`   |
```

Comment:
Do we want to have a torchcodec with cuda column here too?

Reply:
Not sure if I understand correctly. The CPU and CUDA version of torchcodec should have the same expected pytorch version?